.github/workflows/trigger-release.yml

```diff
@@ -0,0 +1,38 @@
+name: Trigger Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish_release:
+    name: Trigger Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Find Current Version
+        id: current_version
+        run: grep version gradle.properties >> $GITHUB_OUTPUT
+      - name: Create Tag
+        env:
+          TAG_NAME: v${{ steps.current_version.outputs.version }}
+        run: |
+          git tag $TAG_NAME
+          git push origin $TAG_NAME
```

Comment:
I might need to adjust how the tag is pushed up, this might not actually trigger the deploy workflow because the `GITHUB_TOKEN` doesn't trigger subsequent GHA workflows to prevent infinite loops